### ISSUE TITLE
SignWriting U+1DA5E + Fill2: who is right?

### ISSUE THREAD

**Issue by Mercury13** [googlefonts/noto-fonts#2260]
### Title 
Sutton SignWriting U+1DA5E + Fill2 + Rotation1…5:
result differs from https://slevinski.github.io/SignWriting_Character_Viewer/SignWriting_Character_Viewer.html#?ui=en&set=uni8&sym=1DA5E

### Font 
NotoSansSignWriting-Regular.ttf

### Where the font came from, and when
Site: this repo
Date: 2021-02-22

### Font Version
2.001

### OS name and version
W10, probably all OS

### Application name and version
Qt 6.1.2, probably all apps

### Issue
  1. Write U+1DA5E + Fill2, or U+1DA5E + Fill2 + Rotation2…5
  2. Does not stick
  3. https://slevinski.github.io/SignWriting_Character_Viewer/SignWriting_Character_Viewer.html#?ui=en&set=uni8&sym=1DA5E  tells that they should stick

### Character data
U+1DA5E SignWriting tongue moves against cheek
U+1DA9B SignWriting fill modifier-2
U+1DAA1 SignWriting rotation modifier-2
...
U+1DAA4 SignWriting rotation modifier-5

### Screenshot
NOTO
![изображение](https://user-images.githubusercontent.com/10164882/155040085-e49747ef-9774-48d8-98c2-9b26368c4485.png)

THAT REFERENCE
![изображение](https://user-images.githubusercontent.com/10164882/155040128-00308dd1-c563-48af-a903-a3894fea6052.png)

**Comment by dscorbett** [comment]
Noto Sans SignWriting supports <U+1DA5E, U+1DA9B> when applied to the base character U+1D9FF SIGNWRITING HEAD, as Unicode specifies.
![𝧿𝩞𝪛](https://user-images.githubusercontent.com/1124347/155047519-f966c276-ed4d-4cb1-9972-0147b0b98414.png)

Note that the SignWriting community disapproves of the Unicode encoding of SignWriting. You can’t assume that their materials are entirely Unicode-compliant. For example, that chart you took a screenshot of intentionally flouts the standard.

See #1972 for the general issue, which is that this font is confused and inconsistent about how combining characters work. This font neither matches Unicode’s recommendations nor anyone else’s.

**Comment by Mercury13** [comment]
Well, now I understand. So Noto needs special backup glyph for this when no head, e.g.
![изображение](https://user-images.githubusercontent.com/10164882/155074014-55aec954-20e3-4165-a79a-e50eaae0c363.png)

**Comment by Slevinski** [comment]
> Note that the SignWriting community disapproves of the Unicode encoding of SignWriting. You can’t assume that their materials are entirely Unicode-compliant. For example, that chart you took a screenshot of intentionally flouts the standard.

For the sake of clarity, the community standards came first.  The [International SignWriting Alphabet 2010](https://steve.signwriting.org/#standard2010) was released in 2010 and has been stable since initial release.  Our character encoding was first published in [January of 2011](https://datatracker.ietf.org/doc/html/draft-slevinski-iswa-2010#appendix-B).  The Unicode proposal was a copy of our proposal  without attribution (N4015 published April of 2011) which eliminated structural markers and number characters.  Subsequent Unicode proposals made [breaking changes](https://www.slideshare.net/StephenSlevinski/sign-writing-in-unicode-8-issues) (N4090 and N4342).  Our concerns were ignored and we left the process in 2012.  In 2015 the UTC informed us that they had approved the encoding for Unicode 8.  After extensive discussions and exhaustive testing, we realized that we couldn't use the official Unicode specification.

The Noto Sans SignWriting font was released in 2020.  I released a package to support this font and the Unicode 8 implementation called [@sutton-signwriting/unicode8](https://github.com/sutton-signwriting/unicode8).  More testing reinforced our decision that we couldn't use the Unicode 8 specification for SignWriting. 

The [reference linked](https://slevinski.github.io/SignWriting_Character_Viewer/SignWriting_Character_Viewer.html#?ui=en&set=uni8&sym=1DA5E) does not flout the standard.  It is the way the encoding was designed to work and we never updated it to be consistent with the Unicode 8 specification.

Our character encoding called [SignWriting in Unicode (SWU)](https://steve.signwriting.org/#standard2017) intentionally flouts the standard. The SignWriting in Unicode (SWU) encoding is isomorphic with our earlier character encoding release in 2012 called [Formal SignWriting in ASCII (FSW)](https://steve.signwriting.org/#standard2012).  These character encodings evolved from earlier character encodings starting in 2004, with a lineage that goes back to 1986.

Our [development continues](https://github.com/sutton-signwriting/font-ttf/issues/4) with SignWriting in Unicode (SWU) with a two-dimensional font.  Our [prototype two-dimensional font](http://signpuddle.net/iswa/#smartfont) was first released in 2014.

**Comment by simoncozens** [comment]
I am making a general point here.

"The community" is often an unhelpful term in this kind of discussion. There are usually multiple language communities at different levels, and many groups and organisations may make claims of authoritative status to speak on behalf of "the community". And so it is sometimes the case that purportedly speaking for "the community" is often a statement about power. Additionally, scripts and languages often develop in ways that that *prescriptive* organisations would rather they did not.

In that context, for the purposes of the Noto project, the question of "who is right" is irrelevant; Noto should implement the Unicode Standard, and not being in the business of trying to adjudicate between different claims.

**Comment by Slevinski** [comment]
I agree 100%.  Noto should implement the Unicode standard.  I am very excited and pleased that the Noto font exists.  I added information to the [SignWriting Wikipedia page](https://en.wikipedia.org/wiki/SignWriting#Accessibility) and I contributed a page about the [Noto Sans SignWriting font on ScriptSource](https://scriptsource.org/cms/scripts/page.php?item_id=entry_detail&uid=wb9v9lchat).  If you want to use the Unicode Standard version of SignWriting, Noto is the only font available.

There is a stark divide between the Unicode community and the SignWriting community.  There is no community writing sign language using the Unicode standard version of SignWriting.  With the Unicode standard, you can write individual symbols but you can not write complete signs.  I have yet to see any sign language written using the Unicode standard version of SignWriting.

If you want to use Sutton SignWriting to write sign language or if you are interested in [machine translation](https://www.sutton-signwriting.io/#machine-learning) between video, sign language, and spoken language, I would avoid the Unicode Standard version of SignWriting and join the SignWriting community.

My claim to speak for the SignWriting community is not based on power, but on competence, encouragement, and support. Over 40 sign languages are written using SignWriting.  Each sign language can have several communities and groups.  We support them all with free resources and possibilities.
